spec/default_spec.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'spec_helper'
+
+describe 'osl-jenkins::default' do
+  cached(:chef_run) do
+    ChefSpec::SoloRunner.new(CENTOS_7_OPTS).converge(described_recipe)
+  end
+  it 'converges successfully' do
+    expect { chef_run }.to_not raise_error
+  end
```

Comment:
Do you intend to fix this test so that it actually runs or leave that for another PR? Looks like some stubbing is missing.